src/main/java/wooteco/subway/PageController.java

```diff
@@ -9,17 +9,18 @@
  */
 @Controller
 public class PageController {
```

Comment:
이 Api 는 Swagger에서 볼 필요가 없을 것 같아요. `@ApiIgnore` 를 사용하면 숨길 수 있습니다 :) 
```suggestion
@ApiIgnore
@Controller
public class PageController {
```